### DISCUSSION TITLE
Solocraft installation

### DISCUSSION THREAD

**Discussion by Argonoize** [azerothcore/azerothcore-wotlk#19540] [Modules]
Hi there,
what is meant with
2) Import the SQL to the right Database (character)
My installation is in ~/azerothcore-wotlk/ and its installed in /modules
whats next?

**Comment by ryankennedy85** [comment]
You’ll need to run the .sql file from the mods ‘data’ folder against your character database using whatever program you chose to edit your MySQL server I.e. Heidisql.
